db/migrate/20170223180546_add_metadata_fields_to_attachments.rb

```diff
@@ -0,0 +1,5 @@
+class AddMetadataFieldsToAttachments < ActiveRecord::Migration
+  def change
+    add_column :attachments, :meta_data_contact_address, :string
```

Comment:
Maybe this should be `print_meta_data_contact_address`, so it's obvious that this is to be used for print only.